File: app/src/main/java/in/woloo/www/review/ViewReviewActivity.java
package in.woloo.www.review;

import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentTransaction;

import android.os.Bundle;
import android.widget.FrameLayout;

import butterknife.BindView;
import butterknife.ButterKnife;
import in.woloo.www.R;
import in.woloo.www.common.CommonUtils;
import in.woloo.www.review.fragments.ViewReviewFragment;
import in.woloo.www.utils.AppConstants;

public class ViewReviewActivity extends AppCompatActivity {

    @BindView(R.id.flFragments)
    FrameLayout flFragments;


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_view_review);
        ButterKnife.bind(this);
        initViews();
    }

    private void initViews() {
        try{
            String review = getIntent().getStringExtra(AppConstants.REVIEW);
            loadFragment(ViewReviewFragment.newInstance(review,""));
        }catch (Exception ex){
             CommonUtils.printStackTrace(ex);
        }
    }

    public void loadFragment(Fragment fragment){
        try{
            FragmentManager fragmentManager = ((AppCompatActivity) this).getSupportFragmentManager();
            FragmentTransaction fragmentTransaction = fragmentManager.beginTransaction();
            //fragmentTransaction.addToBackStack(null);
            fragmentTransaction.replace(R.id.flFragments,fragment);
            fragmentTransaction.commit();
        }catch (Exception ex){
             CommonUtils.printStackTrace(ex);
        }
    }
}

File: app/src/main/java/in/woloo/www/application_kotlin/presentation/activities/loo_discovery/ViewReviewActivity.kt
package `in`.woloo.www.application_kotlin.presentation.activities.loo_discovery

import android.os.Bundle
import android.widget.FrameLayout
import androidx.appcompat.app.AppCompatActivity
import androidx.fragment.app.Fragment
import butterknife.BindView
import butterknife.ButterKnife
import `in`.woloo.www.R
import `in`.woloo.www.application_kotlin.presentation.fragments.loo_discovery.ViewReviewFragment
import `in`.woloo.www.common.CommonUtils
import `in`.woloo.www.utils.AppConstants

class ViewReviewActivity : AppCompatActivity() {
    @BindView(R.id.flFragments)
    lateinit var flFragments: FrameLayout
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_view_review)
        ButterKnife.bind(this)
        initViews()
    }

    private fun initViews() {
        try {
            val review = intent.getStringExtra(AppConstants.REVIEW)
            loadFragment(ViewReviewFragment.newInstance(review, ""))
        } catch (ex: Exception) {
            CommonUtils.printStackTrace(ex)
        }
    }

    fun loadFragment(fragment: Fragment?) {
        try {
            val fragmentManager = (this as AppCompatActivity).supportFragmentManager
            val fragmentTransaction = fragmentManager.beginTransaction()
            //fragmentTransaction.addToBackStack(null);
            fragmentTransaction.replace(R.id.flFragments, fragment!!)
            fragmentTransaction.commit()
        } catch (ex: Exception) {
            CommonUtils.printStackTrace(ex)
        }
    }
}